dockerfiles/node-bench-regression-guard/node-bench-regression-guard

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'optparse'
+
+COMPARISON_THRESHOLD = 2
```

Comment:
Can this be overridden by an argument to the script? Not to say it's necessary, but may come in handy.

Reply:
Added `--comparison-threshold` option which defaults to 2.